src/client/actions/Market.ts

```diff
@@ -35,25 +36,20 @@ export default class Market {
     public async getMarketplaceTemplates(
         page = 1,
         filters: Record<string, string | boolean>,
-        type: string = null
+        type = 'card'
```

Comment:
We shouldn't imply a default type imo

Reply:
![image](https://user-images.githubusercontent.com/11234273/80850087-777de880-8c1a-11ea-8102-98678b484304.png)